src/miniscript/types/extra_props.rs

```diff
@@ -340,6 +382,9 @@ impl Property for ExtData {
             ops_count_static: self.ops_count_static + 1,
             ops_count_sat: self.ops_count_sat.map(|x| x + 1),
             ops_count_nsat: self.ops_count_nsat.map(|x| x + 1),
+            stack_elem_count_sat: self.stack_elem_count_sat,
+            // FIXME: Why not Some(1) ?
+            stack_elem_count_dissat: self.stack_elem_count_dissat,
```

Comment:
This should be None.

Reply:
See : https://github.com/rust-bitcoin/rust-miniscript/pull/148#issuecomment-706815941

`0NOTEQUAL` is added at the end of miniscript. Maybe that is causing the confusion?

Reply:
I don't get your comment ? This was a note to myself on a (very) previous version :) (And ofc Github moved the comment around).